plugins/sep/varnish/varnish.cc

```diff
@@ -26,18 +26,33 @@ void Varnish::setView(ViewInterface::WeakPtr viewWeak) {
   this->viewWeak = viewWeak;
 }
 
+void Varnish::resetView(ViewInterface::WeakPtr viewWeak) {
+  this->viewWeak = viewWeak;
+
+  gdk_threads_enter();
+  GtkWidget *newBox = gtk_vbox_new(false, 0);
+  for (GList *iter = gtk_container_get_children(GTK_CONTAINER(box));
+       iter != nullptr; iter = iter->next) {
+    gtk_widget_reparent(GTK_WIDGET(iter->data), newBox);
+  }
+  box = newBox;
+  viewWeak.lock()->addSideWidget("Varnish", box);
+  gdk_threads_leave();
+}
+
 static void varnish_toggled(GtkToggleButton *button, gpointer varnishP) {
+  (void)button;
```

Comment:
what does this do...? statement seems to have no effect?

Reply:
I didn't write this, but these casts are commonly used to silence the compiler warnings about unused variables. Though as far as I understand it is cleaner to just not give a variable a name in the first place.

Reply:
It is indeed for silencing the compiler warning but not naming the variable is of course cleaner. I changed it accordingly in the commit that will be synced soon.